src/app.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+const operation = process.argv[2];
+const numbers = process.argv.slice(3);
+let res;
+
+if (operation === 'add' && numbers.length > 0) {
+  res = numbers.reduce((acc, cur) => {
+    return Number(acc) + Number(cur);
```

Comment:
Hi)
`return Number(acc) + Number(cur)` - acc is already number(you started from 0 or 1).But if type of cur === string? we will have NaN in the end.We need check every 'cur' - is it number?